composer.json

```diff
@@ -32,7 +32,8 @@
     "require-dev": {
         "cakephp/cakephp-codesniffer": "^4.0",
         "cakephp/authorization": "^2.0",
-        "phpunit/phpunit": "~8.5.0 | ^9.3"
+        "phpunit/phpunit": "~8.5.0 | ^9.3",
+        "cakedc/cakephp-phpstan": "^2.0"
```

Comment:
Is this additional dependency really necessary?

Reply:
@LordSimal Do you have an update on this? Can we resolve this PR?

Reply:
I have removed it.